scripts/ror.js

```diff
@@ -95,7 +98,7 @@ function getRorDisplayHtml(name, url, altNames, truncate = true, addParens = fal
         name = name + '<a href="' + url + '" target="_blank" rel="nofollow" >' + '<img alt="ROR logo" src="https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg" height="20" class="ror"/></a>';
     }
     if (addParens) {
-        name = ' (' + name + ')';
+        name = '(' + name + ')';

```

Comment:
```suggestion
        name = ' (' + name + ')';

```

Line 32 could also be simplified into 

    $(rorElement)[0].previousSibling.data = val.substring(0, index);

Reply:
I now see from https://github.com/IQSS/dataverse/issues/10964 I added the space withing the span. It works but pure semantically not logical.

Reply:
This change was intentional as the spacing is now controlled by CSS (see https://github.com/IQSS/dataverse/blob/efbbd181751efe8fe32986a20a32b3e15be3ead4/src/main/webapp/resources/css/structure.css#L689-L691) and can therefore be changed by custom css as well. I didn't see an easy way to do something more backward compatible. (Another thing to add to the external vocab mechanism (perhaps as part of [#9828](https://github.com/IQSS/dataverse/issues/9828)) - a way to add css as well as js files.)